### PR TITLE
Add support of webpack@2

### PR DESCRIPTION
**Pull request by LoicMahieu**
* Transfert webpack dependency to peerDependecy

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,7 +50,9 @@
   "dependencies": {
     "favicons": "^4.8.3",
     "loader-utils": "^0.2.14",
-    "lodash": "^4.11.1",
-    "webpack": "^1.13.0"
+    "lodash": "^4.11.1"
+  },
+  "peerDependencies": {
+    "webpack": "1 || ^2.1.0-beta"
```
Comment (didi0613) [inline package.json]: Can you please updating to the release webpack v2 here?
@LoicMahieu  Can you please take a look at it? Thank you.

File: package.json [hunk 2]
```diff
@@ -44,13 +44,15 @@
...
   },
   "dependencies": {
     "favicons": "^4.8.3",
     "loader-utils": "^0.2.14",
-    "lodash": "^4.11.1",
-    "webpack": "^1.13.0"
+    "lodash": "^4.11.1"
+  },
+  "peerDependencies": {
+    "webpack": "1 || ^2.2.1"
```
Comment (jchip) [inline package.json]: not sure what's npm's behavior with `1` but should make it `^1` or `^1.13.0` to be more clear.

**Comment by halkeye** [comment]
Looks like all the tests are failing

**Comment by sugarshin** [comment]
Hi,

Any update on this?

File: package.json (hunk 2 above)
Comment (mastilver) [inline package.json]: according to https://semver.npmjs.com/

`1` is like `1.x`, but I agree with you, it should be more explicit

**Comment by davidalee** [comment]
womp womp

**Pull request closed without merging**